kernel/kernel-api/src/main/java/io/delta/kernel/internal/DataWriteContextImpl.java

```diff
@@ -59,7 +61,9 @@ public DataWriteContextImpl(
      * @return fully qualified path of the target directory
      */
     public String getTargetDirectory() {
-        return targetDirectory;
+        // TODO: this is temporary until paths are uniform (i.e. they are actually file system paths
```

Comment:
This will be fixed as part of PR that changes all APIs to use the `path`. Currently we use `paths` some places and `URI` in others. This PR is coming next.